package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@okta/openapi": "2.7.0"
```

Comment:
@VitaliiTytarenko-okta please set it to `2.6.0`, the SDK has not been migrated to `2.7.0` yet.